pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -45,6 +45,7 @@ import (
 const (
 	// backupManifestName is the file name used for serialized BackupManifest
 	// protos.
+	BackupManifestName = "BACKUP_MANIFEST"
 	backupManifestName = "BACKUP_MANIFEST"
```

Comment:
I think we can export `BackupManifestName` instead? I actually think we can avoid exporting it if we use `readBackupManifestFromStore` instead of `readBackupManifest`.